index.html

```diff
@@ -3987,6 +3987,14 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#json-processing"></a>, some software
+applications might not perform full JSON-LD processing.
+<a>Conforming document</a> authors are urged to not use JSON-LD keywords that
+affect document processing in the `@context` value, such as globally setting
+`@base` and `@language`, in order to increase interoperability.
```

Comment:
The key constraint is that the handling of `@base` and `@language` are not (currently) part of the JSON Processing model, and so would result in a deviation in result (as with `@base`) or a "richer" dataset (from inclusion of `@language`) after JSON-LD Processing which would make the two approaches inequivalent.

So, how about this text instead:
> To support consistency across both JSON Processing and
JSON-LD Processing models, the `@base` and `@language` keywords should not
be used within context files nor as inline values within the `@context` array.

Reply:
If we tweak the language here we need it to focus on global settings only -- individual context term mapping definitions in extending context files should be able to use `@language` without issue.

Reply:
I'm not sure it makes sense to sacrifice JSON-LD processing benefits for JSON processors that don't care about JSON-LD.... That feels like watering down the value of JSON-LD, for preserving a value that is not recognized by JSON Processors.

I'd recommend not anchoring the JSON-LD value to the JSON processing experience.

The opposite in fact, JSON-LD processing should have a set of features that are greater than JSON Processors can offer, and we should lean into recommending them... when they make sense.

Reply:
@BigBlueHat not having a concrete change suggestion here is making things difficult to determine if we have consensus on this PR. I don't know if @OR13 is objecting to the original language, your language, or the PR (in general).

At this point, I need to understand if either of your input is blocking or not. Also note that the outcome of https://github.com/w3c/vc-data-model/issues/1264#issuecomment-1712807665 will probably drive more changes in the spec (beyond this PR).

Reply:
@msporny sorry for any confusion. @dlongley's suggestions below (https://github.com/w3c/vc-data-model/pull/1271/files#r1337759360) are great. 👍 to using those instead of my earlier suggestions.